File: comms/push-notifications/users/delete-user/delete-user.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "<username>",
        authToken: "<password>",
    });
    await client.pushNotifications.users.delete("comms_pushnotificationuser_01h9krwprkeee8fzqspvwy6nq8");
}
main();

File: comms/push-notifications/users/delete-user/delete-user.rb
require "twilio"

client = Twilio::Client.new(
  account_id: "<username>",
  auth_token: "<password>"
)

client.push_notifications.users.delete(user_id: "comms_pushnotificationuser_01h9krwprkeee8fzqspvwy6nq8")
